xgi/stats/dinodestats.py

```diff
@@ -248,30 +248,30 @@ def out_degree(net, bunch, order=None, weight=None):
     >>> import xgi
     >>> H = xgi.DiHypergraph([[{1, 2}, {5, 6}], [{4}, {1, 3}]])
     >>> H.nodes.out_degree.asdict()
-    {1: 1, 2: 0, 5: 1, 6: 1, 4: 0, 3: 1}
+    {1: 1, 2: 1, 5: 0, 6: 0, 4: 1, 3: 0}
```

Comment:
was this a mistake that was not caught by our automated checks?

Reply:
Correct. We had in and out-degrees flipped.